src/plugins/data_views/common/data_views/data_view.ts

```diff
@@ -181,7 +181,7 @@ export class DataView extends AbstractDataView implements DataViewBase {
     if (dataViewSpec.fieldAttrs) {
       // removes `count` props (popularity scores) from `fieldAttrs`
```

Comment:
Nit: can we update this comment to reflect that we're removing `customDescription` as well?

Reply:
Thanks!